docs/development_setup.md

```diff
@@ -47,8 +47,9 @@ A minimal database setup for running tests is provided. If the build argument `O
 
 ### Requirements
 
-- Python 3.8+
-- Poetry 1.1.0+
+- Python: 3.8 > 3.11
```

Comment:
Version 3.8 isn't larger than 3.11 ;) I think you wanted to write something like this: `Python: ≥ 3.8 and ≤ 3.11`
Why is this even important? Can't you just say ≥ 3.8?

Reply:
Thanks for the feedback on the version declaration. I will integrate your comments.

Why not > 3.8? Currently in the `pyproject.toml` the Python version is fixed to `>=3.8,<3.11` and not `^3.8`. That is because of the package `scipy`. `poetry` will throw an install error if one tries to install `scipy` with the Python version fixed to `^3.8`. `poetry` suggest to use `>=3.8,<3.11` instead to resolve this issue.